sample/DateTimeParser/DateTimeParser.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="superpower" Version="1.0.2" />
```

Comment:
This needs to be a `ProjectReference` to build against the version of Superpower in this solution.